src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {!comments.length
+      ? <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
this component should know nothing about `comments` being possibly empty, conditional rendering should appear 1 level up